### PR TITLE
fixed issues #4395 & #4343

### PR DESCRIPTION
**Pull request by satvik2131**

### CODE REVIEW

File: android/res/layout/item_localized_name.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
   android:animateLayoutChanges="true"
   android:gravity="center_vertical"
   android:orientation="horizontal">
+
```
Comment (biodranik) [inline android/res/layout/item_localized_name.xml:11]: That change is not needed.

**Review by biodranik — COMMENTED** [review]
Thanks! I like fixes like this one ) @Jean-BaptisteC did you test it?

**Review by satvik2131 — COMMENTED** [review]
@biodranik  Is this PR ok ? Or should I create another one?

**Review by Jean-BaptisteC — APPROVED** [review]
Pixel 6 - Android 13 ✅

Tested on: 
- Tag name, name:en in editor
- numberhouse
- postal code
- phone
- website
- email
- facebook
- instagram
- twitter
- vk
- line

In the future, maybe we can doing same thing in dialog box to create new street in POI ?

**Comment by Jean-BaptisteC** [comment]
@biodranik This PR is waiting for next version planned in next days ?
In the future, if you have PR waiting my tests to push PR and create release, i think it's can be good to add issue or PR in [Release Blockers](https://github.com/organicmaps/organicmaps/milestone/6) when as possible. And maybe give me a delay or date about you think create release, i can try to adapt me, to tests before release :)

**Comment by biodranik** [comment]
@satvik2131 can you please check if the same approach can be used in other dialogs? Editing street name, editing bookmark name, OpenStreetMap.org login name etc 

> In the future, maybe we can doing same thing in dialog box to create new street in POI ?

**Review by biodranik — APPROVED** [review]

**Comment by satvik2131** [comment]
> @satvik2131 can you please check if the same approach can be used in other dialogs? Editing street name, editing bookmark name, OpenStreetMap.org login name etc
> 
> > In the future, maybe we can doing same thing in dialog box to create new street in POI ?

sure

**Comment by Jean-BaptisteC** [comment]
If necessary, i can list all pop up and fragment/activity with fields
In pop-up, it's can be good if we show error message after user have started to fill fields
In some cases, error message is show before user has started to fill fields

**Comment by satvik2131** [comment]
> If necessary, i can list all pop up and fragment/activity with fields
> In pop-up, it's can be good if we show error message after user have started to fill fields
> In some cases, error message is show before user has started to fill fields

Yes , sure you can mention the list of them , so that I can check and integrate them

**Pull request merged**